.travis.yml

```diff
@@ -85,8 +85,14 @@ before_install:
     - tools/check_sdist.py
 
 install:
-    - python setup.py develop
     - ccache --show-stats
+    # Test installing without cython using the sdist
+    - if [[ "$INSTALL_FROM_SDIST" == "1" ]]; then
```

Comment:
Bash scripting is super annoying. I don't understand strings and whatnot.

We can try this syntax, but as I remember it:

- We don't define `INSTALL_FROM_SDIST` in all buildis.
- Even if we did, is "string 0" truthy or falsy?

I'm just bad at shell scripting....

Reply:
> We don't define `INSTALL_FROM_SDIST`

This is true of all the other flags, or at least a good number of them. `$WHATEVER_SOME_VARIABLE_THAT_DOESNT_EXIST` evaluates to nothing/false:

```bash
 $ echo $FOOBARUAU


```

Conversely 0 does *not* evaluate to false:

```bash
 $ FOOBARAU=0
 $ if [[ $FOOBARAU ]]; then echo yes; else echo no; fi
yes
 $ if [[ $NOVARIABLEDOESNTEXIST ]]; then echo yes; else echo no; fi
no
```

Reply:
Thanks @jni. 

I probably authored many of the other shell script tests like `== "1"`. Do you want to keep the convention in this PR and issue an a seperate "cleanup" PR for your preferred syntax?

Reply:
Let's do it right in this PR, then a separate cleanup PR can come later. I don't care too much about consistency with bad conventions. ;) e.g. if some new PR came using `skimage.img_as_float` instead of `util.img_as_float`, I'd want it fixed before merging even though we still have a few of the former in the codebase (hopefully eliminated in my latest PR)